test/blackbox-tests/test-cases/pkg/pin-stanza/update-non-dune-local-pin.t

```diff
@@ -52,8 +52,8 @@ Update the message that gets printed while building foo:
 
 The change to the package is ignored:
   $ dune build 2>&1 | grep -v make
-  echo aaa
-  aaa
+  echo bbb
```

Comment:
Looks like this fixes https://github.com/ocaml/dune/pull/10512. Can you update the comments in this test as they are now incorrect.